addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -223,3 +229,14 @@ def _get_default_mailing_domain(self):
             mailing_domain = expression.AND([mailing_domain, [('phone_sanitized_blacklisted', '=', False)]])
 
         return mailing_domain
+
+    # ------------------------------------------------------
+    # ORM
+    # ------------------------------------------------------
```

Comment:
Remove the section and put the create override after the _compute.

Side note:
Order for methods in models is usually as follows:
- onchange/compute
- create/write/... ORM overrides
- business methods